cmd/agentbasedinstaller/import.go

```diff
@@ -31,32 +37,25 @@ func ImportCluster(ctx context.Context, log *log.Logger, bmInventory *client.Ass
 	}
 	log.Infof("Imported cluster with id: %s", clusterResult.Payload.ID)
 
-	if err := configureClusterIgnitionEndpoint(ctx, bmInventory, clusterConfigDir, clusterResult.Payload.ID); err != nil {
+	if err := updateClusterConfiguration(fsys, ctx, bmInventory, clusterResult.Payload.ID); err != nil {
```

Comment:
I assume `clusterResult.Payload.ID` is the clusterID

Reply:
They are different IINW